src/app.js

```diff
@@ -22,17 +26,31 @@ export function app(app) {
     })
   }
 
-  emit("ready", render(state, view))
+  render()
+
+  emit("init")
 
   return emit
 
-  function render(state, view) {
-    element = patch(
-      root,
-      element,
-      node,
-      (node = emit("render", view)(state, actions))
-    )
+  function render() {
+    if (!locked) {
+      locked = !locked
+
+      requestAnimationFrame(function() {
+        element = patch(
+          root,
+          element,
+          node,
+          (node = emit("render", view)(state, actions))
+        )
+
+        locked = !locked
```

Comment:
This code below produces 2 calls to `render` on app startup with this PR.   Both of the "loaded" event handlers get lumped into one animation frame, so you essentially have the initial render, then a "loaded" render.

Moving the `locked =! locked` after the `emit:loaded` breaks things, and the actions fired by the "loaded" event handlers doesn't trigger another render.  The state still updates, but nothing is rendered, which becomes clear when you click the "increment" button and see the number jump up.

```js
const {h, app} = hyperapp

const mixin = () => {
  return {
    events: {
      loaded: (state, actions) => {
        actions.inc()
      }
    }
  }
}

app({
  state: 1,
  actions: {
    inc: (state) => state +1
  },
  events: {
    loaded: (state, actions) => {
      console.log("loaded")
      actions.inc()
    },
    render: () => console.log("rendering")
  },
  view: (state, actions) => h('div', null, ['Hello ' + state, h('button', {onclick: () => actions.inc()}, "Increment")]),
  mixins: [mixin]
})
```